lib/pluginmanager/command.rb

```diff
@@ -1,8 +1,16 @@
 class LogStash::PluginManager::Command < Clamp::Command
+
+  def initialize(invocation_path, context = {}, parent_attribute_values = {})
+    initialize_gemfile
+    super(invocation_path, context, parent_attribute_values)
+  end
+
```

Comment:
I did this to show one idea of loading sources once, and actually not having to go to each call that need and load the sources because ```rubygems.org```might not be the only one there. If we aim to support private repos, and offline environments, we've to think there are others repositories possible.

I think we should find a solution where sources are loaded once, i can not understand why sources should not be loaded once for all the plugin manager interactions. I will explore other solutions, but at the end loading sources is something that might be better done once, as it's not going to cause any problem, or slow down, just simplifying Gem configuration.

What do you think?